openassessment/xblock/static/js/src/lms/oa_base.js

```diff
@@ -73,6 +76,25 @@ OpenAssessment.BaseView.prototype = {
     },
 
     /**
+     * Install click handlers to set usage key of an XBlock.
+     */
+    setUpUsageID: function() {
+        $(".action--submit").on("click", function(event) {
```

Comment:
Actually, I could not move this to constructor because here we need the DOM to be loaded so as to apply click handler. That happens after the view has rendered, so I would need to call this in the views inheriting this.